_topic_maps/_topic_map.yml

```diff
@@ -52,8 +52,10 @@ Name: Release notes
 Dir: release_notes
 Distros: openshift-enterprise
 Topics:
-- Name: OpenShift Container Platform 4.15 release notes
-  File: ocp-4-15-release-notes
```

Comment:
Yeah, this does not look right. For the newest release, for example 4.17, we would clone the 4.16 release notes and copy them to the 4.17 branch. The `main` branch is not used for release notes but it would be good to preserve a template in main for the release notes.